Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -156,7 +176,7 @@ public static int MaxMemoryCacheSize
             catch (Exception ex)
             {
                 System.Diagnostics.Debug.WriteLine(ex.Message);
-                return null;
+                throw ex;
```

Comment:
Given this is a public method that could be called from anywhere, not just your changes here, not sure how happy I am about just throwing the exception. Would it not be better to just return null anyway, especially as in your use case, that's what you wanted anyway.